app/controllers/coffee_shop_controller.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'sinatra/base'
+require 'json'
+require_relative '../services/coffee_shop_finder_service'
+
+class CoffeeShopController < Sinatra::Base
+  configure do
+    set :protection, except: [:host_authorization]
+    set :show_exceptions, false
+    set :raise_errors, true
+    enable :logging
+  end
+
+  before do
+    content_type :json
+  end
+
+  get '/closest_shops' do
+    content_type 'text/plain'
```

Comment:
Why did you choose this content type?

Reply:
Refer to: [Why does success returns a plain/text and failure returns a json?](https://github.com/Agilefreaks/ruby-interview-starting-point/pull/4#discussion_r1933689378)